src/components/js/EcDynamicField.js

```diff
@@ -0,0 +1,18 @@
+import { discover } from '../../utils/Discoverer.js'
+
+export default {
+  name: 'EcDynamicField',
+  props: {
+    field: {
+      type: String
+    },
+    schema: {
+      type: Object
+    }
+  },
+  computed: {
+    dynamicFields () {
+      return discover(this.field, this.schema)
```

Comment:
Pra poder alterar o objeto `schema`...

```suggestion
      return discover(this.field, cloneDeep(this.schema))
```

Reply:
```js
import cloneDeep from 'lodash.clonedeep'
```